id/news/_posts/2019-08-28-ruby-2-4-7-released.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: news_post
+title: "Ruby 2.4.7 Rilis"
```

Comment:
I prefer "Ruby 2.4.7 Dirilis" or "Ruby 2.4.7 Telah Dirilis" for the title

Reply:
OK. I just updated that to "Ruby 2.4.7 Telah Dirilis".

By the way, how about the old translations that have been production? Should we update it? or just ignore it and focus on the next translation? @kuntoaji

Reply:
I think it's better focus on new translation first for now.

Reply:
OK. Can I merge this now, @kuntoaji?

Reply:
yup